haystack/backends/solr_backend.py

```diff
@@ -48,6 +55,14 @@ def __init__(self, connection_alias, **connection_options):
         self.conn = Solr(connection_options['URL'], timeout=self.timeout, **connection_options.get('KWARGS', {}))
         self.log = logging.getLogger('haystack')
 
+    def get_schema_admin(self):
```

Comment:
Correct: we're only supporting versions of Django which are themselves supported and 1.6 was dropped last April.